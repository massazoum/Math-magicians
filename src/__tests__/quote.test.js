import React from "react";
import '@testing-library/jest-dom/extend-expect'
import renderer from 'react-test-renderer';
import DisplayQuote from '../components/quote';

test('DisplayQuote component renders correctly', () => {
 const component = renderer.create(<DisplayQuote />);
 const tree = component.toJSON();
 expect(tree).toMatchSnapshot();
});

