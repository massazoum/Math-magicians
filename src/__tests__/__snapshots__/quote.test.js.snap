// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayQuote component renders correctly 1`] = `
<div>
  Loading...
</div>
`;
